_drafts/chatbot-draft-one.md

```diff
@@ -0,0 +1,18 @@
+Although a book suggestion chatbot may not seem as exciting and thrilling as a meme bot, it seemed like a perfect topic for two reasons. First off, this topic could apply to real-life situations such as libraries or bookstores and help people find new books. Second, it was simple enough to give room to experiment with coding and broad enough to advance the chatbot and incorporate more code as our knowledge expanded. The first version of the chatbot consisted of simple print statements that did not leave a lot of room for an individualized experience for each user. Later on, we figured out how to not only incorporate user input but also create outputs that would correspond with user input. As our understanding of code grew, the complexity of the chatbot’s code grew with it.
+Creating a chatbot or any type of code can seem intimidating at first, it is not as hard as it looks as long as you have access to the right resources. Throughout my coding experience, I had the help of not only my peers and the Girls Who Code tutorials, but also the help of experienced coders and facilitators that would help with anything from brainstorming to figuring out the code itself. I learned that it was okay to mess up, and I did not need to memorize all of the different commands or script in order to code successfully. Sometimes, it's okay to rely on google to look up the syntax, or even analyze other pieces of code. Within my chatbot, I wanted to implement a randomizing factor that would allow book suggestions to be different for every experience. The line of code that allowed the chatbot to randomize its outputs was actually from a superpower/scientist generator used at the beginning of the Girls Who Code session to introduce us to code. Another thing I learned was to not be afraid to ask questions! I learned just as much through asking specific questions or how to code a certain function as I did through the tutorials and slideshows presented.
+Just like writing an essay, it can help to not only have the goal of your project in mind but to create a roadmap to guide yourself through creating the code. Creating a flowchart of the end goal of the code helps keep everything in order and personally helped a lot with if statements. Another thing to keep in mind when coding is to experiment! You can accomplish so much more or advance your code if you go in without expectations of perfection. Speaking from personal experience, it can seem scary to go through and change some of the code with the fear of messing up the entire project. Saving a copy of the pre-edited code, copy-pasting sections of code, and using hashtags to turn bits of code or ideas still in the works into lines the program skips over made it easier to play around with code, knowing my previous work would not be lost forever.
```

Comment:
Overall, this section is great! But, I would change one thing: starting from "and using hashtags..." I would shorten the sentence a bit because it feels a little long and choppy. Maybe "and using hashtags in which the program skips over the code made it easier to play with." could be a way to shorten it?